selfdrive/car/toyota/interface.py

```diff
@@ -270,7 +290,7 @@ def update(self, c):
     ret.cruiseState.available = bool(self.CS.main_on)
     ret.cruiseState.speedOffset = 0.
 
-    if self.CP.carFingerprint in [CAR.RAV4H, CAR.HIGHLANDERH, CAR.HIGHLANDER] or self.CP.enableGasInterceptor:
+    if self.CP.carFingerprint in [CAR.RAV4H, CAR.HIGHLANDERH, CAR.HIGHLANDER, CAR.RAV4_2019, CAR.COROLLA_HATCH] or self.CP.enableGasInterceptor:
```

Comment:
@wocsor , are those car allowing stop and go without having to press resume?

Reply:
they are, apparently both do